source/Minable.cpp

```diff
@@ -116,7 +116,9 @@ void Minable::Place(double energy, double beltRadius)
 
 	// Generate random orbital parameters. Limit eccentricity so that the
 	// objects do not spend too much time far away and moving slowly.
```

Comment:
The comments in this file provide detail for what things are done the way they are. I'd expect that this new spread variable would be explained at well. This comment notes that the eccentricity is limited (by the `* .6`), but a >1 spread would cause the case that this comment says the multiplication was meant to avoid. So why is spread being used here?